src/components/donate/buttons/AmountButton.js

```diff
@@ -1,15 +1,26 @@
-import { Button, withStyles } from "@material-ui/core";
+import {
+  Button,
+  FormControl,
+  FormGroup,
+  Grid,
+  makeStyles,
+  withStyles,
+} from "@material-ui/core";
 import useData from "../../../hooks/useData";
 
-import React from "react";
+import React, { useState } from "react";
+import { useCampaignConfig } from "../../../hooks/useConfig";
+import { useForm } from "react-hook-form";
+import OtherAmountInput from "../OtherAmount";
+import { useTranslation } from "react-i18next";
 
 const StyledButton = withStyles((theme) => ({
```

Comment:
Ha !

I had some trouble making it work with makeStyles - the styles from MUI and the ones defined here weren't merging.

Since StyledButton is used for both the Other button and the fixed amounts I kind of like this approach - just makes it a little clearer it's shared I think.

Reply:
don't get it, how is this different than exporting the button with a makeStyles/useStyle in it when sharing?

you just do 

const classes = useStyle

and export DaButtonz () { return <Button class={classes.yourFancyStyle}>

Reply:
Right - I know it shouldn't be different, but like I said I had some trouble getting it to work.  Maybe you can just do it ?